src/modules/Builder/features/ActivityItems/ItemConfiguration/OptionalItemsAndSettings/OptionalItemsAndSettings.tsx

```diff
@@ -42,8 +43,27 @@ import { getOptionValue } from './OptionalItemsAndSettings.utils';
 import {
   DEFAULT_OPTION_VALUE,
   ITEMS_TO_HAVE_RESPONSE_OPTIONS_HEADER,
+  ITEMS_WITH_DOWNLOAD_HEADER,
 } from './OptionalItemsAndSettings.const';
 
+const getItemConfigurationHeader = (itemResponseType: ItemResponseType, t: TFunction) => {
+  if (ITEMS_TO_HAVE_RESPONSE_OPTIONS_HEADER.includes(itemResponseType)) {
+    return (
+      <StyledTitleLarge sx={{ mb: theme.spacing(2.4) }}>{t('responseOptions')}</StyledTitleLarge>
+    );
+  }
+
+  if (ITEMS_WITH_DOWNLOAD_HEADER.includes(itemResponseType)) {
+    return (
+      <StyledTitleLarge sx={{ mb: theme.spacing(2.4) }}>
```

Comment:
`theme.spacing` isn't actually needed for `sx`, which calls that function automatically with this shorthand:
```suggestion
      <StyledTitleLarge sx={{ mb: 2.4 }}>{t('responseOptions')}</StyledTitleLarge>
    );
  }

  if (ITEMS_WITH_DOWNLOAD_HEADER.includes(itemResponseType)) {
    return (
      <StyledTitleLarge sx={{ mb: 2.4 }}>
```